resources/views/Reviews/show.blade.php

```diff
@@ -141,11 +142,12 @@
                 <form action="{{ route('reviews.create') }}" method="GET">
                     @csrf
                     <input type="hi
-                    dden" name="" id="">
+                    dden" name="" id="{{$store->id}}">
                     <button type="submit" class="btn-text-danger">
                         <img src="./css/rogo_addreview.jpg" alt="口コミ鉛筆" class="rogo">te
```

Comment:
［SHOULD］
多分画像出てこなくて、viewファイルがフォルダの中にあるので段階を一個上にあげる必要がありそうです。
```suggestion
                        <img src="../css/rogo_addreview.jpg" alt="口コミ鉛筆" class="rogo">
```
あとこのimgタグの後ろに「te」って入っているので消してください～

Reply:
今これみて気づいたんですけど、cssのフォルダの中に画像おいてるんですねw